pyproject.toml

```diff
@@ -10,35 +10,28 @@ description = "Python and C interfaces for logging"
 readme = "README.txt"
 authors = [{name = "Anthony Tuininga", email = "anthony.tuininga@gmail.com"}]
 keywords = ["logging"]
-license = {text = "BSD License"}
+license = "BSD-3-Clause"
+license-files = ["LICENSE.txt"]
 classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Intended Audience :: Developers",
-    "License :: OSI Approved :: BSD License",
     "Natural Language :: English",
     "Operating System :: OS Independent",
     "Programming Language :: C",
     "Programming Language :: Python",
     "Topic :: Software Development :: Libraries :: Python Modules",
     "Topic :: Utilities",
 ]
-requires-python = ">=3.9"
+requires-python = ">=3.11"
```

Comment:
Is this needed?

Reply:
As per the title, build wheels for python 3.11-3.14...
But if you want something starting from 3.9 or 3.10, we can change it...